tgui/packages/tgui/interfaces/GPS.tsx

```diff
@@ -0,0 +1,214 @@
+/**
+ * @file
+ * @copyright 2025
+ * @author FlameArrow57 (https://github.com/FlameArrow57)
+ * @license ISC
+ */
+
+import { BooleanLike } from 'common/react';
+import {
+  Box,
+  Button,
+  Collapsible,
+  Divider,
+  LabeledList,
+  NumberInput,
+  Section,
+  Stack,
+} from 'tgui-core/components';
+
+import { useBackend } from '../backend';
+import { Window } from '../layouts';
+
+interface GPSInfo {
+  src_x: number;
+  src_y: number;
+  track_x: number;
+  track_y: number;
+  tracking: BooleanLike;
+  trackable: BooleanLike;
+  src_name: string;
+  distress: BooleanLike;
+  gps_info: GPSTrackable[];
+  imp_info: GPSTrackable[];
+  warp_info: GPSTrackable[];
+}
+
+interface GPSTrackable {
+  name: string;
+  ref: string;
+  x: number;
+  y: number;
+  z_info: string;
+  distress: BooleanLike | null;
+}
+
+const gpsTooltip =
+  'Each GPS is coined with a unique four digit number followed by a four letter identifier.';
+
+export const GPS = () => {
+  const { act, data } = useBackend<GPSInfo>();
+  const {
+    src_x,
+    src_y,
+    track_x,
+    track_y,
+    tracking,
+    trackable,
+    src_name,
+    distress,
+    gps_info,
+    imp_info,
+    warp_info,
+  } = data;
+
+  return (
+    <Window title="GPS" width={460} height={610} theme="ntos">
+      <Window.Content>
+        <Section title={`GPS Device ${src_name}`}>
+          <Stack>
+            <Stack.Item>
+              <LabeledList>
+                <LabeledList.Item label="Coordinates">
+                  {src_x}, {src_y}
+                </LabeledList.Item>
+                <LabeledList.Item label="Identifier">
+                  <Button
+                    onClick={() => act('change_identifier')}
+                    tooltip={gpsTooltip}
+                  >
+                    {src_name.slice(5, src_name.length)}
+                  </Button>
+                </LabeledList.Item>
+                <LabeledList.Item label="Coords">
+                  x:{' '}
+                  <NumberInput
+                    value={track_x}
+                    width={'3'}
```

Comment:
```
                    width={3}
```
or
```
                    width="3"
```
Both should resolve to the same, one may not be okay because of typing, if so just use the other.

Reply:
It stops having default width of 3 with these changes

Reply:
Weird. `width="3"` should be entirely comparable to `width={'3'}`, just less shit.